index.js

```diff
@@ -8,8 +8,51 @@ const http = require('http'),
         The time is: ${rightNow}
       </p>
     </body>
+ 
+      <body>
+   <div style="position:  absolute; bottom: 5px; background-color: white">
+<h2> Characters in The Lord of The Rings </h2>
+ <ul>
+   <li>   <b> Frodo Baggins </b>  </li>
```

Comment:
I see, I'm not really quite sure if I had read that somewhere, or I just reverted back to High school HTML class...